config/sync/field.field.config_pages.stanford_basic_site_settings.su_site_url.yml

```diff
@@ -12,7 +12,7 @@ field_name: su_site_url
 entity_type: config_pages
 bundle: stanford_basic_site_settings
 label: 'Site URL'
-description: 'To redirect all traffic to a single url, input the stanford.edu url here.'
+description: 'To redirect all traffic to a single url, input the stanford.edu url here.<br><li>The domain supplied must be the same domain as the base url.</li>'
```

Comment:
I would check with Rebecca/Jacob and/or someone from the UX team on the wording here. This wording does not make sense to me, and we've already experienced people FUBARing their sites due to misunderstanding what this setting does.

Reply:
Probably needs a link to the SNOW form to request a hostname.